servicetalk-opentracing-inmemory/src/main/java/io/servicetalk/opentracing/inmemory/AbstractInMemorySpanBuilder.java

```diff
@@ -52,13 +53,13 @@ protected AbstractInMemorySpanBuilder(String operationName, int maxTagSize) {
 
     @Override
     public final InMemorySpanBuilder asChildOf(SpanContext parent) {
-        addReference(References.CHILD_OF, parent);
+        addReference(CHILD_OF, parent);
         return this;
     }
 
     @Override
     public final InMemorySpanBuilder asChildOf(Span parent) {
-        addReference(References.CHILD_OF, parent.context());
+        addReference(CHILD_OF, parent.context());
```

Comment:
Do we need similar public methods to simplify adding `FOLLOWS_FROM` references?

Reply:
This method is overriden from `io.opentracing.SpanBuilder` to refine/specialize the return type. I don't think it is necessary to add additional methods for `followsFrom` but we can do this in a followup PR if necessary.